packages/vscode-extension/src/project/project.ts

```diff
@@ -372,6 +376,52 @@ export class Project implements Disposable, MetroDelegate, ProjectInterface {
     this.deviceSession?.onActiveFileChange(filename, followEnabled);
   }
 
+  public async getCommandsCurrentKeyBinding(commandName: string) {
```

Comment:
can we extract this method from project.ts ? It seems to be very loosely related and we already have a ton of code in this file

Reply:
@kmagiera I agree but It seems that the best solution would be to create a new interface for utilities like that for example: 
- reportIssue(): Promise<void>;
- openDevMenu(): Promise<void>;
- openFileAt(filePath: string, line0Based: number, column0Based: number): Promise<void>;
- movePanelToNewWindow(): void;

I'll create another PR for that at it is out of scope for this one. What do you think?

Reply:
Seconded, it'd be nice to have it in utils/ or common/. Refactor can be done later.